nautobot/docs/installation/migrating-from-netbox.md

```diff
@@ -327,178 +322,17 @@ The `broadcast` will be derived from the `prefix_length` and will be that of the
-
-NetBox:
-
-```python
-Prefix.objects.filter(prefix__net_contains=str(instance.prefix))
-```
-
-Nautobot:
-
-```python
-Prefix.objects.net_contains(instance.prefix)
-```
-
-##### net_contains_or_equals
-
-*Returns target networks that contain or equal the source network.*
-
-NetBox:
-
-```python
- Aggregate.objects.filter(prefix__net_contains_or_equals=str(self.prefix))
-```
-
-Nautobot:
-
-```python
-Aggregate.objects.net_contains_or_equals(self.prefix)
-```
-
-##### net_equals
-
-*Returns target networks that equal the source network.*
-
-NetBox:
-
-```python
-Prefix.objects.filter(prefix=str(self.prefix))
-```
-
- Nautobot:
-
-```python
-Prefix.objects.net_equals(self.prefix)
-```
-
----
-
-!!! important
-	The following lookups only apply to `IPAddress` objects.
-
-##### net_host
-
-*Returns target IP addresses that match the host of the source IP address.*
-
-!!! note
-	This custom lookup does not have a manager method equivalent. This pattern should use the `host` field for filtering.
-
-NetBox:
-
-```python
-IPAddress.objects.filter(vrf=self.vrf, address__net_host=str(self.address.ip))
-```
-
-Nautobot:
-
-```python
-IPAddress.objects.filter(vrf=self.vrf, host=self.host)
-```
-
-##### net_host_contained
-
-*Returns target IP addresses that are contained by the source prefix.*
-
-NetBox:
-
-```python
-IPAddress.objects.filter(address__net_host_contained=str(self.prefix))
-```
-
-Nautobot:
-
-```python
-IPAddress.objects.net_host_contained(self.prefix)
-```
-
-##### net_in
-
-*Returns target IP addresses that are members of any of the provided source prefixes.*
-
-NetBox:
-
-```python
-IPAddress.objects.filter(address__net_in=[n1, n2, n3])
-```
-
-Nautobot:
-
-```python
-IPAddress.objects.net_in([n1, n2, n3])
-```
-
-##### ip_family
-
-*Returns target addresses matching the specified family.*
-
-NetBox:
-
-```python
-IPAddress.objects.filter(address__family=family)
-```
-
-Nautobot:
-
-```python
-IPAddress.objects.ip_family(family)
-```
+Nautobot 1.0.x and 1.1.x did not support the custom lookup expressions that NetBox supported for membership queries on IPAM objects (such as `Prefix.objects.filter(prefix__net_contained="10.0.0.0/24")`), but instead provided an alternate approach using model manager methods (such as `Prefix.objects.net_contained("10.0.0.0/24")`).
 
----
+In Nautobot 1.2.0 and later, both model manager methods and custom lookup expressions are supported for this purpose, but the latter are now preferred for most use cases and are generally equivalent to their counterparts in NetBox.
 
-!!! important
-	The following lookups apply to all network objects.
 
 ##### net_mask_length
 
 *Returns target addresses matching the source address prefix length.*
 
 !!! note
-	This custom lookup does not have a manager method equivalent. This pattern should use the `prefix_length` field for filtering.
+	The NetBox filter net_mask_length should use the `prefix_length` field for filtering.
```

Comment:
Perhaps just remove this note, I don't think it adds much compared to the worked examples immediately below.